### PR TITLE
I238-f8-remove board view

### PR DESCRIPTION
**Pull request by mishaone**
* Removed references to board view in UG and GS doc
* Added one troubleshooting section
* Fixed on broken image link

### CODE REVIEW

**Comment by mishaone** [comment]
Hi @rkratky - tests fine locally, just tagging you for a quick look if you have time.

**Comment by Preeticp** [comment]
@mishaone, as per our discussion yesterday, I mentioned, I was already working on the work item sections in UG, and requested you to take up sections in the GS to be updated for this. I suppose there was some misunderstanding here.
There are some topics in this PR which I am Ok to merge, but in other topics I want to just comment out the board section as they are generic and would be valid once it is reintroduced in the next sprints and the rest of the info I am working on.
@rkratky, before this is merged, can we please discuss if this can be simply addressed by resolving a merge conflict I will encounter once my changes for this are in.

**Comment by mishaone** [comment]
Oops, sorry, Preeti. It slipped my mind. Feel free to discard my PR and do
your changes. I'll just quickly redo the valid ones from my changes. There
are separate commits for the GS and UG but I'm not at my machine and
probably wouldn't be confident cherry picking between our work anyway.

File: docs/topics/modules/closing_workitem.adoc [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-After implementing and reviewing your code changes, update the relevant work item to track the work done.
-
-. Click the *Plan* tab and then *Board* to return to the Planner Board view.
-. Drag the work item to the *Resolved* state.
-+
-image::resolved_wi.png[Resolved work item]
-
-Resolving a work item communicates to the team that the task being tracked is complete. Once your task is reviewed and approved you can drag it to the *Closed* state.
-
-Congratulations! You have now created your first quickstart project in {osio}, made changes to your project code, committed the changes to GitHub, and published the new version of your project, and updated the work item to reflect your work.
```
Comment (rkratky) [inline docs/topics/modules/closing_workitem.adoc]: I will miss this.

**Comment by rkratky** [comment]
Oops, apologies. I just realized you probably didn't want me to merge this yet, right? Should I revert it?

>  can we resolve this by simply resolving the merge conflict I will encounter once my changes for this are in?

@Preeticp I hope so :)

File: docs/topics/modules/closing_workitem.adoc (hunk 1 above)
Comment (mishaone) [reply docs/topics/modules/closing_workitem.adoc]: Aww. I'll add it back in, hopefully with enhancements, for the next minor PR.

**Comment by Preeticp** [comment]
Nvm, @mishaone, @rkratky.

File: docs/topics/modules/closing_workitem.adoc (hunk 1 above)
Comment (rkratky) [reply docs/topics/modules/closing_workitem.adoc]: I was kidding!

**Pull request merged**